website/docs/ParameterizeTests.md

```diff
@@ -0,0 +1,167 @@
+---
+id: parameterize-tests
+title: Parameterize Tests
+---
+
+You can simply parameterize tests on a test level, e.g.:
+
+```ts title=example.spec.ts
+const people = ['Alice', 'Bob']
+describe('my tests', () => {
+    for (const name of people) {
+        it(`testing with ${name}`, async () => {
+            // ...
+        })
+    }
+})
+```
+
+## Passing Environment Variables
```

Comment:
I feel like this part could be part of our best practices guide instead, what do you think?